### PR TITLE
Added a student avatar

### PR DESCRIPTION
**Pull request by Hackerbone**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link]( https://hackerbone.github.io) 

- Fixes #<Issue Number>

## Description / Changes
Added Student: Sitaraman Subramanian
Ps: For the preview I didn't upload all the images. Sorry for inconvinience
## Screenshots if any:
screenshot 2018-11-05 at 8 57 11 pm

- - - - - - - - - - - -

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -81,3 +81,7 @@
   github: shreeshjha
   image: shreeshjha.png
   
+- name: Sitaraman Subramanian
+github: Hackerbone
```
Comment (sk9331657) [inline _data/students.yml]: Fix the indentation and provide the correct preview link.

File: _data/students.yml [hunk 2]
```diff
@@ -81,3 +81,7 @@
   github: shreeshjha
   image: shreeshjha.png
   
+- name: Sitaraman Subramanian
+  github: Hackerbone
+  image: student_nick.png
```
Comment (abishekvashok) [inline _data/students.yml]: It should be sitaraman.jpg

Comment (Hackerbone) [reply _data/students.yml]: Ohkay

Comment (Hackerbone) [reply _data/students.yml]: Done

**Comment by Hackerbone** [comment]
Rectified all the changes.. :-)

File: _data/students.yml (hunk 2 above)
Comment (abishekvashok) [reply _data/students.yml]: Still, the same.

**Comment by Hackerbone** [comment]
The changes are rectified now

File: _data/students.yml (hunk 2 above)
Comment (Hackerbone) [reply _data/students.yml]: Corrected it

**Pull request merged**